tests/cpydiff/types_complex_parsing.py

```diff
@@ -0,0 +1,14 @@
+"""
+categories: Types,complex
+description: uPy's complex() accepts certain incorrect values that CPython rejects
```

Comment:
Please use "MicroPython" instead of "uPy", because not everyone knows that the latter is shorthand.

(Would be good to change remaining uPy's in the cpydiff directory, eventually.)